src/SmartComponents/Systems/Systems.js

```diff
@@ -169,12 +164,18 @@ const Systems = () => {
 
     const getEntities = useGetEntities(fetchSystems, apply, {}, history, applyMetadata, applyGlobalFilter);
 
-    const remediationDataProvider = () => remediationProviderWithPairs(
-        removeUndefinedObjectKeys(selectedRows),
-        prepareRemediationPairs,
-        transformPairs,
-        remediationIdentifiers.advisory
-    );
+    const remediationDataProvider = () => {
+        setRemediationLoading(true);
```

Comment:
If we are setting the remedationLoading here, is it needed in `showRemediationModal`?

Reply:
@gabipodolnikova thank you for the review, actually it is present there as well. Git did not show that line becuase it was present before this PR and Git did not detect any change.